distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Improved: [#18632] Land ownership and construction rights are now shown on top of the water.
 - Improved: [#20951] Activate OpenRCT2 window after using native file dialog on macOS.
+- Fix: [#20616] Confirmation button in the track designer's quit prompt now says "OK".
```

Comment:
Fixes in the changelog should list the problem, rather than the solution. Also, we use proper quotation marks instead of typewriter quotes.

```suggestion
- Fix: [#20616] Confirmation button in the track designer’s quit prompt has the wrong text.
```